tests/manual/clipboard/ClipboardExtImageTest.java

```diff
@@ -0,0 +1,208 @@
+        } else {
+            testStatus.setText("PASSED (No spare temporary files found)");
+            testStatus.setTextFill(Color.GREEN);
+        }
+    }
+
+    private Scene createScene() {
+        Label warn = new Label();
+        warn.setText("This test refers to a Windows-specific issue.");
+
+        Label instructions = new Label();
+        instructions.setText(
+            "Follow these steps to test the scenario:\n" +
+            " 1. Open your web browser, find a random picture and copy it to system's Clipboard\n" +
+            " 2. Click on \"Test - Read Clipboard\" button\n" +
+            "If image was fetched from Clipboard properly, it will show in the scroll pane below.\n" +
+            "After the test is run there should be no extra temporary files left by JFX in the TMP dir."
+        );
+        instructions.setMinHeight(90.0);
+
+        Label testStatusHeader = new Label();
+        testStatusHeader.setText("Test status:");
+
+        testStatus.setText("...");
+
+        Button testButton = new Button("Test - Read Clipboard");
+        testButton.setOnAction((ActionEvent) -> {
+            try {
+                testTemporaryFilesLeftover();
+            } catch (Exception e) {
+                testStatus.setText("FAILED (Caught exception: " + e.getMessage() + ")");
+                testStatus.setTextFill(Color.RED);
+            }
+        });
+
+        Button openTmpButton = new Button("Open TMP dir");
+        openTmpButton.setOnAction((ActionEvent) -> {
+            try {
+                if (isWindows()) {
+                    Runtime.getRuntime().exec(new String[]{"explorer.exe", new File(System.getProperty("java.io.tmpdir")).getAbsolutePath()});
+                }
+            } catch (Exception e) {
+                Alert alert = new Alert(AlertType.ERROR);
+                alert.setTitle("Failed to open TMP dir");
+                alert.setContentText("TMP dir failed to open: caught Exception " + e.getMessage());
+                alert.show();
+            }
+        });
+
+        if (!isWindows()) {
+            openTmpButton.setDisable(true);
+        }
+
+        clipboardImageView.setPickOnBounds(true);
+        clipboardImageView.setPreserveRatio(true);
+
+        ScrollPane pane = new ScrollPane(clipboardImageView);
+        pane.setVbarPolicy(ScrollPane.ScrollBarPolicy.ALWAYS);
+        pane.setHbarPolicy(ScrollPane.ScrollBarPolicy.ALWAYS);
+        pane.setFitToHeight(true);
+        pane.setFitToWidth(true);
+
+        VBox box = new VBox(warn, instructions, testStatusHeader, testStatus, testButton, openTmpButton, pane);
+        box.setAlignment(Pos.TOP_LEFT);
+        box.setSpacing(5.0);
+        box.setFillWidth(true);
+        box.setVgrow(pane, Priority.ALWAYS);
+        box.setPadding(new Insets(15.0));
+
+        return new Scene(box, 800, 600);
+    }
+
+    @Override
+    public void start(Stage primaryStage) throws Exception {
+        try {
+            /*if (!isWindows()) {
```

Comment:
This test can be only Windows specific.
I would enable this check and get rid of `isWindows()` check everywhere.

Reply:
Done